TripKit/AddOns/InterApp/TKInterAppCommunicator.h

```diff
@@ -8,16 +8,23 @@
 
 #import <UIKit/UIKit.h>
 
+@import MapKit;
+
 @class TKSegment;
 
 typedef enum {
   TKInterAppCommunicatorITunesAppIDGoCatch  = 444439909,
   TKInterAppCommunicatorITunesAppIDIngogo   = 463995190,
   TKInterAppCommunicatorITunesAppIDLyft     = 529379082,
   TKInterAppCommunicatorITunesAppIDOla      = 539179365,
-  
 } TKInterAppCommunicatorITunesAppID;
 
+typedef NS_ENUM(NSInteger, TKInterAppCommunicatorMapDirectionMode) {
+  TKInterAppCommunicatorMapDirectionModeWalking,
+  TKInterAppCommunicatorMapDirectionModeDriving,
+  TKInterAppCommunicatorMapDirectionModeCycling,
+  TKInterAppCommunicatorMapDirectionModeDefault
+};
```

Comment:
I made some similar changes in #67. Better to base this branch on #67, so that we don't double up. In particular you can then use `TKTurnByTurnMode` and get rid off `TKInterAppCommunicatorMapDirectionMode`.